playbooks/roles/minishift/tasks/main.yml

```diff
@@ -58,4 +58,4 @@
 
 # Initialize minishift
 - import_tasks: init_minishift.yml
-  when: minishift_exist.stdout != ""
+  when: (minishift_profile_status.stdout|lower == "does not exist") or minishift_profile_status.rc != 0
```

Comment:
why parens around one and not the other if you are going to use them?

Reply:
Let me fix that.